# Idea: Does the implementation of new bus route improves circulation?
## _$H_0$_ : The commute time after implementation is longer or equal to the time of pre-implementation.  _$t_0$_ >= _$t_1$_
## _$H_a$_ : The communte time after implementation is shorter to the time of pre-implementation.  _$t_0$_ < _$t_1$_
## significance level  $\alpha=0.05$


In [31]:
import pandas as pd
import numpy as np
from math import sqrt

In [32]:
time=pd.read_csv("times.txt", header=None)
time.head()

0
0  31.622239
1  32.821376
2  30.229101
3  31.413766
4  39.010550

In [33]:
z = (36 - time.mean())/(6/sqrt(len(time)))
print("z value: {}". format(z[0]))

z values: 2.55639718616667


- Given $\alpha=0.05$, then the corresponding critical value for the z-test at 5% significance level would be 2.  
- Since z value is equal to 2.556 which is larger than 2, it falls outside of the 95% inclusion under normal distribution. Also, this also means that the results are 2 sigmas far away from the original mean.
- Hence, we can reject null hypothesis at 5% significance level and conclude that the communte time after implementation is shorter to the time of pre-implementation.